.github/workflows/appimage.yml

```diff
@@ -102,7 +102,7 @@ jobs:
 
     - name: Build AppImage (Latest)
       if: github.event_name == 'workflow_dispatch' || github.event_name == 'push'
-      run: sudo -u testbot bash -lc 'cd viam-cartographer && make BUILD_CHANNEL="latest" appimage'
```

Comment:
Following the logic here:
1. If we are doing a manual workflow run or pushing a PR, this "Build AppImage (Latest)" will be triggered
2. make appimage is run
3. package_release_module is run
4. If there is a new tag  ----> an rc or stable appimage will be produced
6. If there is no tag -----> a latest appimage will be produced

Please correct me if any of that is wrong. 

Additionally, this logic chain means that a workflow dispatch manual run will regenerate and appimages rc, stable and latest. Is this intended behavior? Or do we want to add a check to see if an appimage exists before creating a new one?

Reply:
Yep, that's right!

Hmm. Would a manual run regenerate the appimages today as well? If so, I'd rather fix that in a separate ticket if we do fix it.

Reply:
Yes, they would so a separate ticket would be fine. Note: we'd still want the PR appimage to regenerate as the code might have been updated but the others ones should not. The only reason to rerun a workflow on a stable/latest/rc is if the appimage failed due to a flaky test.

Reply:
not sure I follow this convo, are we saying we want a separate ticket after this to make a manually run appimage workflow will generate a `latest` appimage? As far as I can tell this current workflow will do only that already

Reply:
I think Jeremy was saying we want a ticket to avoid generating the same appimage again if the appimages have already been generated, then the workflow is manually run again.

But it's a good point that we may actually want to keep that behavior, at least for PR's.